h/services/search_index/_queue.py

```diff
@@ -35,8 +35,6 @@ def add(self, annotation_id, tag, schedule_in=None, force=False):
     def add_all(self, annotation_ids, tag, schedule_in=None, force=False):
         """Queue a list of annotations to be synced to Elasticsearch."""
```

Comment:
Still think it's time to document this with params

Reply:
Done: https://github.com/hypothesis/h/pull/6347/commits/4952ad8bbde373dbe917dce7e6cee9e5b8aec912 It's a shame that every public method in `Queue` has a corresponding forwarding method in `SearchIndexService` though. Which of the two methods do you put the docstring on? I've gone for putting them on `Queue` near to the actual code